docs/make.bat

```diff
@@ -9,7 +9,7 @@ if "%SPHINXBUILD%" == "" (
 )
 set SOURCEDIR=.
 set BUILDDIR=_build
-set SPHINXPROJ=minos_microservice_networks
+set SPHINXPROJ=minos_microservice_common
```

Comment:
I think this would be referred to networks. I am right? 🤔

Reply:
Nice catch. I think this is what is breaking the test's pipeline :sweat: